core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala

```diff
@@ -601,10 +600,6 @@ class GroupMetadataManager(brokerId: Int,
               throw new IllegalStateException(s"Unexpected unload of active group $groupId while " +
                 s"loading partition $topicPartition")
           }
-
-          if (!shuttingDown.get())
```

Comment:
I am not sure, but it is weird, right? Why would we be loading during shutdown, and if we are, it seems useful knowing about it in the logs.